### DISCUSSION TITLE
Multiple Twilio InBox on Same Number

### DISCUSSION THREAD

**Discussion by Stephen-Lobo** [chatwoot/chatwoot#2987] [Q&A] [answered]
Is it possible to have multiple Twilio Inboxes on Same Twilio number ?

We have one Global Twilio number which is used by different Departments. We would like to have multiple inboxes to have different access level / team members per department. Is this possible ?

When we tried to set up second Twilio Inbox, chatwoot displays some authentication error.

**Comment by muhsin-k** [comment, marked as answer]
@Stephen-Lobo At the moment, it is not possible.
